hooks/RoboFile.php

```diff
@@ -55,6 +55,17 @@ public function setupSkeleton()
         $config->extra->{"enable-patching"} = 'true';
         $config->extra->{"patches"} = new \stdClass();
         file_put_contents('composer.json', json_encode($config));
+
+        // Create a directory for our artifacts.
+        $this->taskFilesystemStack()
```

Comment:
I think that it is time to add a CHANGELOG so it's easier to figure out what to change when updating the image tag, right?

Reply:
Yes, my goal is to get to a point where I can tag `0.1.0` and do a changelog. My thinking is that as far as semver goes, we consider any change that requires updating the copied `config.yml` file to be a minor change, then hopefully a major change when we get to 1.0.0.